tests/integration/test_ourlogs.py

```diff
@@ -73,7 +73,7 @@ def envelope_with_otel_logs(ts: datetime) -> Envelope:
 def timestamps(ts: datetime):
     return {
         "sentry.observed_timestamp_nanos": {
-            "stringValue": time_within(ts, expect_resolution="ns", precision="s")
+            "stringValue": time_within(ts, expect_resolution="ns")
```

Comment:
This was incorrect, before, the `observed_timestamp_nanos` is set from the received timestamp, which is derived from the current time in Relay, not an input.